tests/jobs/test_scheduler_job.py

```diff
@@ -1010,7 +1010,7 @@ def test_process_dags_not_create_dagrun_for_subdags(self):
         scheduler._process_task_instances = mock.MagicMock()
         scheduler.manage_slas = mock.MagicMock()
 
-        scheduler._process_dags(self.dagbag, [dag] + dag.subdags, [])
+        scheduler._process_dags([dag] + dag.subdags, [])
```

Comment:
Maybe make the second argument optional? Instead of passing an empty iterable.

Reply:
In another PR I deletes the tis_out parameter completely.  This parameter is also required because this method returns values using this list. Unfortunately, we do not have an assertion in the tests. In real code is always required.